purchases/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -184,6 +185,11 @@ class BillingWrapperTest {
         } returns Intent()
     }
 
+    @After
+    fun tearDown() {
+        clearAllMocks()
+    }
```

Comment:
Same as before: moved the `clearAllMocks()` call to an `@After` method since having it in `setup` potentially means we left some mocks from other tests.